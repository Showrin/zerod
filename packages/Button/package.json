{
  "name": "@zerod/button",
  "version": "0.0.4",
  "publishConfig": {
    "access": "public"
  },
  "main": "lib/index.js",
  "module": "src/index.js",
  "dependencies": {
    "react": "latest",
    "react-dom": "latest",
    "styled-components": "latest"
  },
  "peerDependencies": {
    "react": "^17.0.0",
    "react-dom": "^17.0.0",
    "styled-components": "^5.0.0"
  }
}
