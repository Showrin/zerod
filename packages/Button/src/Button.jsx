import React from "react";
import styled from "styled-components";

// styled.button`
//   background: red;
//   color: #fff;
//   border-radius: 4px;
//   cursor: pointer;
//   font-size: 1rem;
//   padding: 9px 36px;
// `

const Button = ({ children }) => <button>{children}</button>;

export default Button;
