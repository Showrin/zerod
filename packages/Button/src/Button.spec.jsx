import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Button from '.';

describe('Button', () => {
  test('renders button correctly', () => {
    const tree = renderer.create(<Button>{'Test'}</Button>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
