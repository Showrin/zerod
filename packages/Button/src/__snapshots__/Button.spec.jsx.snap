// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders button correctly 1`] = `
<button
  className=""
>
  Test
</button>
`;
